### PR TITLE
handle http request error from estuary in shuttle

### PR DESCRIPTION
**Pull request by en0ma**

### CODE REVIEW

File: cmd/estuary-shuttle/main.go [hunk 1]
```diff
@@ -1354,20 +1355,22 @@ func (s *Shuttle) createContent(ctx context.Context, u *User, root cid.Cid, fnam
 
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		return 0, err
+		return 0, errors.Wrap(err, "failed to Do content create request")
```
Comment (alvin-reyes) [inline cmd/estuary-shuttle/main.go:1358]: Best if we just put the "method name" on the error message.

```
"Failed to Do createContent"
```

It's good for log searching too (if we are using `vim` or `nano`).

File: cmd/estuary-shuttle/main.go [hunk 2]
```diff
@@ -1354,20 +1355,22 @@ func (s *Shuttle) createContent(ctx context.Context, u *User, root cid.Cid, fnam
 
 	resp, err := http.DefaultClient.Do(req)
 	if err != nil {
-		return 0, err
+		return 0, errors.Wrap(err, "failed to Do createContent")
```
Comment (gmelodie) [inline cmd/estuary-shuttle/main.go:1358]:
```suggestion
		return 0, errors.Wrap(err, "failed to createContent")
```

File: cmd/estuary-shuttle/main.go [hunk 3]
```diff
@@ -1354,20 +1355,22 @@ func (s *Shuttle) createContent(ctx context.Context, u *User, root cid.Cid, fnam
...
 	}
-
 	defer resp.Body.Close()
 
-	var rbody util.ContentCreateResponse
-	if err := json.NewDecoder(resp.Body).Decode(&rbody); err != nil {
-		return 0, err
+	if resp.StatusCode != 200 {
```
Comment (gmelodie) [inline cmd/estuary-shuttle/main.go:1362]:
```suggestion
	if resp.StatusCode != http.StatusOK {
```

File: cmd/estuary-shuttle/main.go [hunk 4]
```diff
@@ -1404,20 +1407,22 @@ func (s *Shuttle) shuttleCreateContent(ctx context.Context, uid uint, root cid.C
...
-		return 0, err
+		return 0, errors.Wrap(err, "failed to do shuttle content create request")
 	}
-
 	defer resp.Body.Close()
 
-	var rbody util.ContentCreateResponse
-	if err := json.NewDecoder(resp.Body).Decode(&rbody); err != nil {
-		return 0, err
+	if resp.StatusCode != 200 {
```
Comment (gmelodie) [inline cmd/estuary-shuttle/main.go:1414]:
```suggestion
	if resp.StatusCode != http.StatusOK {
```

**Review by gmelodie — APPROVED** [review]
Good stuff!

**Pull request merged**